index.js

```diff
@@ -0,0 +1 @@
+exports.formatCurrency = require('./helpers/currency-format')
```

Comment:
No semicolon, with semicolon? I don't know anymore 

<img src="https://user-images.githubusercontent.com/1079135/36145840-e7d0ee1a-10b2-11e8-9939-c63b068a5bc0.gif" width="100%">

Reply:
Once this is merged, we can add eslint to this repo and be consistent!